bottlecap/src/proxy/interceptor.rs

```diff
@@ -167,10 +167,11 @@ async fn invocation_next_proxy(
             .and_then(|v| v.to_str().ok())
         {
             if let Ok(trigger) = IdentifiedTrigger::from_slice(&intercepted_bytes) {
+                let mut tasks = tasks.lock().await;
                 appsec
                     .lock()
                     .await
-                    .process_invocation_next(request_id, &trigger);
+                    .process_invocation_next(request_id, &trigger, &mut tasks);
```

Comment:
This might not work with how we support streaming

Reply:
Request streaming isn't really a thing in Lambda, though, is it?

Reply:
I meant the #771 when we are proxying

Reply:
How would this have to be changed then?